### ISSUE TITLE
Add {post_terms} token to OpenAI custom prompt

### ISSUE THREAD

**Issue by stevejburge** [TaxoPress/TaxoPress#2123]
Can we allow AI options such as OpenAI to use only the existing terms on the site?

This could possibly be a custom prompt: 
"Choose the most appropriate {tags} from the following content: '{content}'"

We have had some users who want something a little more flexible than an exact match, but are also nervous of OpenAI hallucinating new terms.

**Comment by ojopaul** [comment]
@stevejburge Can't user already customize their prompt? Also, the request can take only a single prompt or do you mind elaborating on what you want me to do?

1. Do you want us to change current default prompt?
OR
2. Do you want us to add {tags} token so people can use it in their custom prompt?

**Comment by stevejburge** [comment]
@ojopaul If this can be solved by a custom prompt that's fine by me. Do you know how a custom prompt could be restricted only to existing terms on the site?

**Comment by ojopaul** [comment]
I've added `{post_terms}` token to acceptable token in OpenAI custom prompt.

I tested it with the following custom prompt and it works:

> Choose the most appropriate tags between these '{post_terms}' from the following content: '{content}'

1. I use `{post_terms}` instead of suggested `{tags}` so as not to give impression this only works for post tags alone as it should work for both tags, categories and other custom taxonomies
2. You'll need to update the documentation with this new token and sample prompt https://taxopress.com/docs/openai-prompts/
@stevejburge
